lib/data-fetcher.js

```diff
@@ -33,7 +43,7 @@ const USER_AGENT = ['Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) Appl
 function fetchMetadataDescription(url) {
   return new Promise((resolve, reject) => {
     fetchWithUA(url)
-      .then(response => response.text())
+      .then(response => DOMPurify.sanitize(response.text()))
```

Comment:
@ithinkihaveacat I actually think is better to make sure cheerio gets a clean html just in case.